### PR TITLE
fix | Security issue fix for Dependabot bump.

### PR DESCRIPTION
**Pull request by ChetanT-System**
## Description
- Security issue fixed.
- Dependabot bump version fixes.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: modules/sde-common/pom.xml
Comment (amoldashwant) [inline modules/sde-common/pom.xml:1]: Update year

Comment (ChetanT-System) [reply modules/sde-common/pom.xml:1]: Updated

File: modules/sde-common/pom.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <!--
 /********************************************************************************
  * Copyright (c) 2022 BMW GmbH
- * Copyright (c) 2022, 2023 T-Systems International GmbH
- * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2024 T-Systems International GmbH
```
Comment (sachinargade123) [inline modules/sde-common/pom.xml:5]: Remove the space between years => 2022,2024

Comment (ChetanT-System) [reply modules/sde-common/pom.xml]: Removed

**Review by sachinargade123 — APPROVED** [review]
LGTM

**Comment by amoldashwant** [comment]
LGTM

**Review by amoldashwant — APPROVED** [review]
LGTM

**Comment by adityagajbhiye9** [comment]
@ChetanT-System Please check dependency.

**Comment by ChetanT-System** [comment]
> @ChetanT-System Please check dependency.

@adityagajbhiye9  Updated !

**Comment by ChetanT-System** [comment]
@almadigabor Please review and approve the PR.

**Pull request merged**